src/components/ControlPanel.vue

```diff
@@ -733,6 +734,7 @@
       <AddEvidenceDialog
         v-if="evidenceData"
         v-model="addEvidenceDialog"
+        :evidenceData="evidenceData"
```

Comment:
@devknight216 - This looks like a duplicate of the line below and doesn't exist in the `props` of `AddEvidenceDialog` so please remove it

Reply:
@dacoaster, just removed that. :) Please merge it on your side.